portindex2json/portindex2json.tcl

```diff
@@ -64,7 +64,58 @@ proc remove_extra_braces {text} {
     return [regsub -all {[\{\}]} $text ""]
 }
 
-set fd [open [lindex $argv 0] r]
+proc print_usage {} {
+    puts stdout "\nUsage: /path/to/tclsh portindex2json.tcl <path-to-portindex> \[-info key=vlaue\]"
```

Comment:
```suggestion
    puts stdout "\nUsage: /path/to/tclsh portindex2json.tcl <path-to-portindex> \[-info key=value\]"
```

Reply:
Maybe add the second line of usage, listing `--help` instead of `<path-to-portindex> [-info key=value]`.